Test fails with error: encoding declaration in Unicode string (<string>, line 0)

Travis CI tests for Python 2.7 fails with
```
AssertionError: example in file discovery_v1.py failed with error: encoding declaration in Unicode string (<string>, line 0)
```

--- a/test/test_examples.py
+++ b/test/test_examples.py
@@ -52,7 +52,7 @@ def test_examples():
 
         try:
             service_file = open(example).read()
-            exec(re.sub('# coding=utf-8', '', service_file), globals())
+            exec(re.sub('# coding[:=]\s*utf-8', '', service_file), globals())
         except Exception as e:
             assert False, 'example in file ' + name + ' failed with error: '\
                           + str(e) + '\n' + traceback.format_exc()

